pandas/core/reshape/tile.py

```diff
@@ -476,9 +476,14 @@ def _format_labels(bins, precision, right=True,
     if right and include_lowest:
         # we will adjust the left hand side by precision to
         # account that we are all right closed
-        v = adjust(labels[0].left)
-
-        i = IntervalIndex([Interval(v, labels[0].right, closed='right')])
+        #
+        # We cannot access labels[0].left because that
+        # calls Interval(left, right, closed='right')
```

Comment:
I am not clear what the issue is here, can you show a full example

Reply:
removed.